bears/cmake/CMakeLintBear.py

```diff
@@ -21,11 +21,16 @@ class CMakeLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
+                         cmake_filter: str=None,
                          cmakelint_config: path=''):
         """
         :param cmakelint_config: The location of the cmakelintrc config file.
+
```

Comment:
This blank line is not necessary. LGTM otherwise 😃

Reply:
Done!